recipes/tamkin/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "tamkin" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd96f36589a86a72fc9362a7141a6c731d51f76294a54eed225646df280c8b67
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.0
+    - matplotlib >1.1
+    - molmod >=1.3.2
```

Comment:
@tovrstra There is currently an issue with rendering the recipe - or more over recipes in general - once this is fixed we should both have access to the conda recipe and can modify it. By the way should I also add you to the conda-forge recipes for molmod and yaff?

Reply:
Thanks but I don't think it is necessary. The repository already has a conda recipe, which is just slightly different, to interact well with the CI. Having two recipes would cause more confusion and it is also extra work to keep the same file in sync in both places.

Reply:
The recipes are already there https://github.com/conda-forge/molmod-feedstock and https://github.com/conda-forge/yaff-feedstock so it is more just a question of adding you as a maintainer. In terms of comparison between your own channel and a community channel I guess the advantage of conda-forge is the automation with its integration in pypi and the stability by using a unified build environment.